rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/utils/Zipper.java

```diff
@@ -160,7 +170,9 @@ public static void unzip(InputStream is, File outFile) throws IOException {
                         entryContainer.mkdirs();
                     }
                     if (!entryFile.isDirectory()) {
-                        Zipper.ZIP.unzipEntry(zis, new FileOutputStream(entryFile));
+                        FileOutputStream outputStream = new FileOutputStream(entryFile);
```

Comment:
don't we need to close this outputStream ?

Reply:
Same remark